### PR TITLE
fix(FEC-7872): Player stuck in case of multirequest is not fully loaded

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes
* Solution for Android Chrome which requires user gesture on the specific video element (part of https://github.com/kaltura/playkit-js/pull/193)
* Do not wait to `sourceselected` event before ad playing, play ad immediately, see https://github.com/kaltura/playkit-js-ima/pull/45  

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [x] Docs have been updated

### CODE REVIEW

File: src/player.js [hunk 1]
```diff
@@ -436,14 +436,17 @@ export default class Player extends FakeEventTarget {
    */
   play(): void {
     if (this._engine) {
-      this._playbackMiddleware.play(this._play.bind(this));
+      this._playbackMiddleware.play(() => this._play());
     } else if (this._loadingMedia) {
       // load media requested but the response is delayed
       Player._prepareVideoElement();
-      this._eventManager.listenOnce(this, CustomEventType.SOURCE_SELECTED, () => this.play());
+      if (this.config.plugins.ima) {
```
Comment (OrenMe) [inline src/player.js]: Make this condition more generic and add it as a generic config.
The application can decide this, in this case if we have IMA or in future another ad plugin, we can set it in Kaltura-Player, or application that uses Kaltura player can even force this condition.

**Comment by OrenMe** [comment]
@yairans please fix title and description

File: src/player.js (hunk 1 above)
Comment (OrenMe) [reply src/player.js]: let's make this callback name also more generic, as any middleware chain might want to use this. `_playAfterAsyncMiddleware` sounds short :satisfied: and concise

**Comment by yairans** [comment]
@OrenMe  actually no any condition is needed, see my last commit

**Pull request merged**